src/app/zeko/circuits/rollup_state.ml

```diff
@@ -10,7 +10,7 @@ let push_events_checked state actions =
       Zkapp_account.Actions.push_events_checked state actions )
 
 module type Action_state_type = sig
-  include SnarkType
+  type t = F.t [@@deriving snarky]
```

Comment:
You don't need this, and shouldn't have this.

Reply:
The same as with imt, I need it if I want to serialise it. I don't see why would it be bad.

Reply:
You shouldn't, but you already can through `typ`.